benefits/core/context_processors.py

```diff
@@ -16,7 +16,7 @@ def _agency_context(agency):
     return {
         "eligibility_index_url": agency.eligibility_index_url,
         "help_templates": unique_values(
-            [v.help_template for v in agency.eligibility_verifiers.all() if v.help_template is not None]
+            [v.help_template for v in agency.eligibility_verifiers.filter(active=True) if v.help_template is not None]
```

Comment:
I wonder if it would be better to have a helper on the agency like `active_verifiers()` to encapsulate this logic? How many places is this used?

Reply:
It's used in 2 places, namely here and on the form. I think it's a good idea to have the helper method so will add that

Reply:
_(and a couple tests)_ :wink:

Reply:
Added a unit test for the new property here: https://github.com/cal-itp/benefits/pull/1970/commits/d79f06c186a8491f2576e3f5a8588fff3ea66740#diff-ccbb4cf80e90dac80960d6a62148f66f688b0fa7e1149a3e5106d614ec56863dR320-R336

I think that's sufficient, but let me know if I'm missing some variation that should be covered